Automattic/wp-calypso
# accessible-focus: Extract into package, re-write in TypeScript, test, and handle deprecated APIs

#### Changes proposed in this Pull Request

* Extract `accessible-focus` into a package
* Rewrite it in TypeScript
* Add tests
* Handle deprecated APIs

#### Testing instructions



* Open the app and navigate using the keyboard. Ensure that the `accessible-focus` classname is added to body.
* Open the app and navigate using your mouse. Ensure the classname is not added.
* Ensure the classname is removed when you switch to mouse navigation.
